docs/RELEASE-PROCESS.md

```diff
@@ -0,0 +1,87 @@
+# Release process
+
+## Determine the new version
+Extension versions may not have suffixes like "-rc1."
+We are incrementing patch versions to differentiate release candidate builds.
+
+For example, the 1.0 release will start with a build at version 1.0.0.
+Version 1.0.0 is, at first, a release candidate.
+If changes are needed, we move to the next build at version 1.0.1.
+Version 1.0.1 is also a release candidate at first.
+If no changes are needed, 1.0.1 becomes the first public release of 1.0.
+
+## Set the new version throughout
+You must update:
+
+- config.js
+- package.json
+- public/manifest.json
+
+## Add a changelog section
+Here's the [changelog](changelog.md).
```

Comment:
Should we rather use a top-level `CHANGELOG.md`?

Reply:
:shrug: that seems valid too. anyone have a preference? same for this doc, docs/ or top level? also all caps or lowercase? >_>